cmd/build/local_test.go

```diff
@@ -0,0 +1,55 @@
+package build
+
+import (
+	"strings"
+	"fmt"
+	"testing"
+)
+
+type mockContext struct {
+	App     string
+	Env     string
+	Account string
+}
+
+func (c mockContext) GetApp() string {
+	return c.App
+}
+
+func (c mockContext) GetEnvironment() string {
+	return c.Env
+}
+
+func (c mockContext) GetAccount() string {
+	return c.Account
+}
+
+func TestProvider_Local(t *testing.T) {
+
+	ctx := mockContext{
+		App:     "my-app",
+		Env:     "dev",
+		Account: "123456789",
+	}
+
+	provider, err := GetProvider("local")
+	if err != nil {
+		t.Fail()
+	}
+	artifacts, err := provider.ProvideArtifacts(ctx)
+	if err != nil {
+		t.Fail()
+	}
+	if artifacts == nil {
+		t.Fail()
+	}
+	if artifacts[0].FilePath != "build.sh" {
+		t.Fail()
+	}
+	t.Log(artifacts[0].FileContents)
+
+	image := fmt.Sprintf("IMAGE=%v.dkr.ecr.us-east-1.amazonaws.com/%v:0.1.0", ctx.Account, ctx.App)
```

Comment:
Since artifacts[0] is the build script, if you take my advice above and double-quote the assignment in here, you need to add the quotes here as well so the string comparison will succeed.